Makefile

```diff
@@ -11,13 +11,7 @@ build:
 
 .PHONY: test
 test:
-	go test -v `go list ./...`
-
-.PHONY: coverage
-coverage:
-	goverage -coverprofile=cover.out `go list ./...`
-	go tool cover -func=cover.out
-	@rm -rf cover.out
+	go test -covermode atomic -coverprofile cover.out -v `go list ./...`
```

Comment:
Just curious if we still need the `go list ./...` part. A simple `go test -covermode atomic -coverprofile cover.out -v ./...` would do. What do you reckon?

Reply:
You were right. `./...` did it as well